app/src/debug/kotlin/io/homeassistant/companion/android/developer/catalog/HAComposeCatalogScreen.kt

```diff
@@ -255,6 +260,102 @@ private fun LazyListScope.textStyles() {
     }
 }
 
+private fun LazyListScope.input() {
+    catalogSection(title = "Input") {
+        Column(
+            modifier = Modifier.fillMaxWidth(),
+        ) {
+            var value1 by remember { mutableStateOf("") }
+            var value2 by remember { mutableStateOf("") }
+            var value3 by remember { mutableStateOf("") }
+            var value4 by remember { mutableStateOf("") }
+            var value5 by remember { mutableStateOf("") }
+
+            HATextField(
+                value = value1,
+                onValueChange = { value1 = it },
+                trailingIcon = {
+                    if (value1.isNotBlank()) {
+                        IconButton(onClick = { value1 = "" }) {
+                            Icon(
+                                imageVector = Icons.Default.Close,
+                                contentDescription = null,
+                            )
+                        }
+                    }
+                },
+            )
+            HATextField(
+                value = value2,
+                onValueChange = { value2 = it },
+                placeholder = {
+                    Text(
+                        text = "Placeholder",
+                        style = HATextStyle.UserInput.copy(color = Color.Unspecified),
```

Comment:
Assuming this is for the preview only, but otherwise: making this look like normal text made me want to select it to clear it.

Reply:
This part is where I need UI helps from the design team.